### ISSUE TITLE
Design detailed wireframes

### ISSUE THREAD

**Issue by jenniferthibault** [18F/fec-alpha#54]
1. **Goal:** Use visual identity system and patterns to design high fidelity wireframes
2. **Completion Criteria:** When designs have been compared to style guide and match, and when FEC gives them a thumbs up
3. **Tasks to completion:**
4. [ ] Page 1? 
5. [ ] Page 2?
6. **Dependencies:** #52 

@noahmanger i'm a bit unsure of how we're thinking of structuring this one (which only points to how good of an exercise this issue template is). Are there specific pages involved? Edits most welcome

**Comment by noahmanger** [comment]
Hmmm yeah. So we have these pages:
1. Home page
2. Registration Options
3. Checklist
4. Knowledge center node
5. Calendar

Do we want to do high-fidelity comps for each?

**Comment by jenniferthibault** [comment]
I started with items that could help figure out the visual hierarchy.
- Registration Options
- Checklist
- Knowledge Center Item

They look a little naked without a menu, so I think bringing that in from #55 will help

![registration options](https://cloud.githubusercontent.com/assets/11636908/9041543/87b42ef8-39d6-11e5-9a28-9cdfbe966fa3.png)

![checklist](https://cloud.githubusercontent.com/assets/11636908/9041550/8e6c0130-39d6-11e5-95b3-b865eafec138.png)

![knowledge center item](https://cloud.githubusercontent.com/assets/11636908/9041551/91809d86-39d6-11e5-820b-576703b6ca9b.png)

As I designed each new page, I found elements that need to be tweaked throughout, and new opportunities for patterns and visualization, so I think that stuff will continue to evolve.

I think these pages demonstrate icon styles and typography styles really well, and how & where each would be used.
